services/galley/test/integration/API/MLS.hs

```diff
@@ -197,17 +248,17 @@ aliceInvitesBob SetupOptions {..} = withSystemTempDirectory "mls" $ \tmp -> do
 
   -- create a group
 
-  groupId <-
-    toBase64Text . unGroupId
+  (groupId, conversation) <-
+    first (toBase64Text . unGroupId)
       <$> if createConv
         then do
           conv <-
             responseJsonError =<< postConvQualified (qUnqualified alice) defNewMLSConv
               <!! const 201 === statusCode
           liftIO $ case cnvProtocol conv of
-            ProtocolMLS mlsData -> pure (cnvmlsGroupId mlsData)
+            ProtocolMLS mlsData -> pure (cnvmlsGroupId mlsData, cnvQualifiedId conv)
             p -> assertFailure $ "Expected MLS conversation, got protocol: " <> show (protocolTag p)
-        else pure "test_group"
+        else pure ("test_group", error "No conversation created")
```

Comment:
How about using `assertFailure` instead of returning this pair?

Reply:
This is not a failure, though. We could return a `Maybe` on the second component, but whatever, this is just a test code, what does it matter if we crash with error vs get an assertion failure? It can only happen if there's a bug in the test itself.

Reply:
Agreed it's not a big deal. It can for sure stay this way.